.project-keeper.yml

```
@@ -7,4 +7,4 @@ sources:
       - jar_artifact
 excludes:
   - "E-PK-CORE-18: Outdated content: '.github/workflows/ci-build.yml'"
-  - "E-PK-CORE-18: Outdated content: '.github/workflows/release_droid_prepare_original_checksum.yml'"
+ciBuildRunnerOS: ubuntu-20.04
```

We could add the comment here? 
```suggestion
ciBuildRunnerOS: ubuntu-20.04 # UDFs fail with "VM error: Internal error: VM crashed" on ubuntu-latest
```

---

But please rate this proposal as _optional_.